planner/core/optimizer.go

```diff
@@ -88,6 +88,7 @@ const (
 	flagPrunColumnsAgain
 	flagPushDownSequence
 	flagResolveExpand
+	flagEliminateApply
```

Comment:
We usually keep the the flag order same with rule order. I think we'e better move flagEliminateApply before flagEliminateProjection